typescript/cli/src/config/multisig.ts

```diff
@@ -84,6 +92,19 @@ export async function createMultisigConfig({
       result[chain] = lastConfig;
       continue;
     }
+    if (Object.keys(sdkContractAddressesMap).includes(chain)) {
+      const reuseCoreConfig = await confirm({
+        message: 'Use existing Hyperlane validators for this chain?',
+      });
+      if (reuseCoreConfig) {
+        result[chain] = {
```

Comment:
Should we just leave it out of `result` in this case? We can still merge it during the deployment step, but allows the SDK to be the source of truth

Reply:
Makes sense. I was thinking if the user selected the chain, they'll expect to see the config in their yaml file. But. I'm don't feel strongly about either approach.